### DISCUSSION TITLE
Unable to add row to an existing collection , Add Row Button "Greyed"

### DISCUSSION THREAD

**Discussion by dev-vick05** [AntlerVC/firetable#1114] [Support / Q&A] [answered]
I'm a new Rowy user and I've created a new table using my existing Firestore collection and sub-collection. The table populates the data correctly but I'm unable to add new rows and the button greyed in colour. Been going through Rowy documentation but unable to figure out the issue.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @dev-vick05, this is probably due to the fact that your table is of type 'collection group'. I found a similar discussion that might help: https://github.com/rowyio/rowy/discussions/1072

**Reply by dev-vick05** [reply]
Hi @BeeBombshell. Yes it is a collection group. Thanks for your help.

Another query. After the table is populated, I've unable to view the sub-collections data associated with the collection. Is there any extra setup needed ?

Thank you

**Reply by BeeBombshell** [reply]
@dev-vick05, you have the option to either see an existing sub-collection as a top-level table (provide the collection path while creating a table) or you can create a sub-collection as a field, which is documented [here](https://docs.rowy.io/field-types/sub-table).
